tomb/Cargo.toml

```diff
@@ -31,6 +31,8 @@ env_logger = "^0.10"
 async-trait = "^0.1"
 hex = "^0.4"
 tomb-common = { path = "../tomb-common", version = "^1", features = ["banyan-api"] }
+open = "5.0.0"
+base64-url = "2.0.0"
```

Comment:
See other comment on version pinning.

I also believe we have base64 already present as a dependency which supports web safe Base64 encoding as well and should likely be used instead of this.